app/views/signs/show.html.haml

```diff
@@ -21,7 +21,7 @@
         = add_vocab_button('vocab_sheet.item.add', class: 'add_to_sheet')
     .videos.small-12.medium-5
       .video-container
-        = videojs_rails sources: { mp4: @sign.video }, class: "main_video video normal", controlsList:"nodownload", controls: true, preload: "auto", loop: true
+        = videojs_rails sources: { mp4: @sign.video }, class: "main_video video normal", controlsList: "nodownload", controls: true, preload: "auto", loop: true
```

Comment:
These still show the play button now with the controls added back on. To meet the story requirements we'll need to remove that but leave the other controls on.